lib/hydra/derivatives/services/retrieve_source_file_service.rb

```diff
@@ -0,0 +1,8 @@
+module Hydra::Derivatives
+  class RetrieveSourceFileService
+
+    def self.call(object, source_name)
```

Comment:
Add comment explaining that this default implementation simply assumes that `+source_name+` corresponds to the method you can call on `+object+` to retrieve the source file.